docs/sources/flow/reference/components/loki.process.md

```diff
@@ -382,6 +382,30 @@ following key-value pair to the set of extracted data.
 username: agent
 ```
 
+{{% admonition type="note" %}}
+Due to a limitation of the upstream jmespath library, any string that contains
+a hyphen `-` needs to be wrapped in quotes so that it's not considered a
+numerical expression.
+
+Otherwise, you might get errors like:
+`Unexpected token at the end of the expression: tNumber`
+{{% /admonition %}}
+
+In this case, you can use one of two syntaxes to circumvent this issue:
```

Comment:
@tpaschalis which issue is being dealt with here? The one in the note?

Reply:
Yes. Is it clear, or should I also include it on the note?